src/InnerHit/NestedInnerHit.php

```diff
@@ -84,15 +75,11 @@ public function getQuery()
     }
 
     /**
-     * @param BuilderInterface $query
-     *
-     * @return $this
+     * @param Search $query
      */
-    public function setQuery(BuilderInterface $query)
+    public function setQuery(Search $query = null)
```

Comment:
Please remove default value from setter. It makes no sense to have setter like this.

Should be: 
```php
public function setQuery(Search $query)
{
    $this->query = $query;

    return $this;
}
```